packages/ruleset/test/schema-resolution.test.js

```diff
@@ -0,0 +1,48 @@
+const { testRule } = require('./utils');
+
+describe('Spectral behavior assumptions', () => {
```

Comment:
If we want to contain any Spectral behavior assumption tests to this suite, which might be a good idea, we could rename this file and use it for any future tests too